specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -4396,6 +4399,15 @@
       "type": "string",
       "description": "The tenant ID.",
       "x-ms-parameter-location": "client"
+    },
+    "top": {
```

Comment:
Based on the description, this should be marked as `required: false`.  Is that correct?

Reply:
You are right! Fixed.